pkg/garden-dev/lib/docket.hoon

```diff
@@ -17,23 +17,56 @@
         license=(unit cord)
     ==
   ::
-  ++  finalize
+  ++  finalize-0
     |=  =draft
-    ^-  (unit docket)
+    ^-  (unit docket-0)
     ?~  title.draft  ~
     ?~  info.draft  ~
     ?~  color.draft  ~
     ?~  version.draft  ~
     ?~  website.draft  ~
     ?~  license.draft  ~
-    =/  href=(unit href)
+    =/  href-0=(unit href-0)
       ?^  site.draft  `[%site u.site.draft]
       ?~  base.draft  ~
       ?^  glob-http.draft
         `[%glob u.base hash.u.glob-http %http url.u.glob-http]:draft
       ?~  glob-ames.draft
         ~
       `[%glob u.base hash.u.glob-ames %ames ship.u.glob-ames]:draft
+    ?~  href-0  ~
+    =,  draft
+    :-  ~
+    :*  %1
+        u.title
+        u.info
+        u.color
+        u.href-0
+        image
+        u.version
+        u.website
+        u.license
+    ==
+  ::
+  ++  finalize
```

Comment:
Wondering if we just want a parameterized `+finalize`. You hand is the version, and it branches where it needs to. Instead of having two separate but largely-duplicate implementations.